sys-account/Makefile

```diff
@@ -27,5 +27,12 @@ this-print: ## print
 	#$(MAKE) srv-print
 
 ## This builds everything
-this-build: ##
-	$(MAKE) flu-gen-lang
+this-build: #go-gen-dep
```

Comment:
go-generate to generate rpc service and definitions for both clients and servers.